wpilibcIntegrationTests/src/FRCUserProgram/cpp/mockds/MockDS.cpp

```diff
@@ -0,0 +1,90 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "MockDS.h"
+
+#include <iostream>
```

Comment:
Is iostream actually used? You need to include stdint.h.

Reply:
Whoops no its not. Removing.